akka-contrib/src/main/scala/akka/contrib/throttle/TimerBasedThrottler.scala

```diff
@@ -1,305 +1,301 @@
-/**
- * Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
- */
-
-package akka.contrib.throttle
-
-import scala.concurrent.duration.{ Duration, FiniteDuration }
-import scala.util.control.NonFatal

```

Comment:
In case you wonder what happened here. There's a trailing '\r' after "NonFatal" that was removed together with the import which seems to make this file being treated as binary.